### DISCUSSION TITLE
monorepo 文件夹路径优化

### DISCUSSION THREAD

**Discussion by yoyo837** [hellof2e/quark-design#254] [Ideas]
建议 
https://github.com/hellof2e/quark-design/tree/main/packages/quark 
重命名为
https://github.com/hellof2e/quark-design/tree/main/packages/quarkd

容易找错目录

**Comment by xxxsf** [comment]
可以的，之前是有这个打算。回头处理下。

**Comment by yoyo837** [comment]
https://github.com/hellof2e/quark-design/tree/main/packages/quarkd
和
https://github.com/hellof2e/quark-design/tree/main/packages/quark-design

更倾向哪一个? 我蹭个PR ?

**Reply by xxxsf** [reply]
都可

**Reply by xxxsf** [reply]
你晚了一步，已修改后合并到主分支了

**Reply by yoyo837** [reply]
👌
